slash/react/src/List/ContentItemMono/ContentItemMono.tsx

```diff
@@ -0,0 +1,75 @@
+import { useMemo, type ReactNode } from "react";
+import { getComponentClassName } from "../../Form/core";
+import { ContentItemMonoSize } from "./constants";
+
+type ContentItemMonoProps = {
+  children: ReactNode;
+  className?: string;
+  classModifier?: string;
+  secondaryText?: string;
+  tertiaryText?: string;
+  leftElement?: ReactNode;
+  size?: ContentItemMonoSize;
+  isDisabled?: boolean;
+  isLeftElementCentered?: boolean;
+  hasStick?: boolean;
+};
+
+export const ContentItemMono = ({
+  children,
+  className,
+  classModifier,
+  secondaryText,
+  tertiaryText,
+  leftElement,
+  size = ContentItemMonoSize.M,
+  isDisabled = false,
+  isLeftElementCentered = false,
+  hasStick = false,
+}: ContentItemMonoProps) => {
+  const componentClassName = useMemo(() => {
+    let newClassModifier = classModifier ?? "";
+
+    if (isDisabled) {
+      newClassModifier += " disabled";
+    }
+
+    newClassModifier += ` ${size}`;
+
+    return getComponentClassName(
+      className,
+      newClassModifier,
+      "af-content-item-mono",
+    );
```

Comment:
```typescript
const classMoifiers = [classModifier, size];

if (isDisabled) {
  classModifiers.push("disabled");
}

return getComponentClassName(
  className,
  classModifiers.filter(Boolean).join(" "),
  "af-content-item-mono",
);

Reply:
Resolved in : https://github.com/AxaFrance/design-system/pull/436/commits/ec43b3f2597f6534cdd448628c2c73b44c7fc178